sbol/property.py

```diff
@@ -767,6 +767,8 @@ def add(self, uri):
         # implemented.
         raise Exception('Should ReferencedObject.add be deprecated?')
         if self._sbol_owner is not None:
+            if self._rdf_type not in self._sbol_owner.properties:
+                self._sbol_owner.properties[self._rdf_type] = []
```

Comment:
This lines shouldn't be needed for two reasons: 1. setting up the sbol owner properties is handled in the constructor, and B. because the whole method is effectively stubbed out by the `raise Exception` line.

Reply:
This has been removed.  I think I needed it when I was working with the outdated tree.